apps/examples/e2e/tests/test-ui.spec.ts

```diff
@@ -1,10 +1,109 @@
-import test from '@playwright/test'
+import test, { expect } from '@playwright/test'
 import { setup } from '../shared-e2e'
 
-test.describe('ui', () => {
+test.describe('mobile ui', () => {
 	test.beforeEach(setup)
-
-	test('mobile style panel opens and closes when tapped', async ({ isMobile }) => {
+	test('mobile style panel opens and closes when tapped', async ({ isMobile, page }) => {
+		test.skip(!isMobile, 'only run on mobile')
+		expect(await page.isVisible('.tlui-style-panel')).toBe(false)
```

Comment:
I'd extract most of these hard coded strings into variables / consts. Unless we only use it once it will make maintaining this code much easier as it will mean we only have to rename it once. It also makes the code a bit more nicer to read (`isVisible(StylePanel)` feels a bit better than `isVisible(.tlui-style-panel)`)

I think it's also worthwile to avoid using css selectors in most cases. It's not what user sees. So in most cases I'd use things like text selectors, role selectors, and if none of those are available I'd probably go for a data test id.

Reply:
Could also change this to:
```
await expect(page.getByTestId(StylePanel)).toBeVisible()
```

These kinds of assertions feel better to read to me.